PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+<!-- Thanks for contributing to the openHAB project! -->
+<!-- Please describe the goal and effect of your PR here. -->
+<!-- Pay attention to the below notes and to *the guidelines* for this repository. -->
+<!-- Feel free to delete any comment lines in the template (starting with "<!--"). -->
+
+<!-- TITLE -->
+
+<!-- Please provide a PR summary in the *Title* above, according to the following schema: -->
+<!-- - If related to one specific addon: Mention the addon shortname in square brackets -->
+<!--   e.g. "[exec]", "[netatmo]" or "[tesla]" -->
+<!-- - If the PR is work in progress: Add "[WIP]" -->
+<!-- - Give a short meaningful description in imperative mood -->
+<!--   e.g. "Add support for device XYZ" or "Fix wrongly handled exception" -->
+<!--   for a new add-on/binding: "Initial contribution" -->
+<!-- Examples: -->
+<!-- - "[homematic] Improve communication with weak signal devices" -->
+<!-- - "[timemachine][WIP] Initial contribution" -->
+<!-- - "Update contribution guidelines on new signing rules" -->
+
+<!-- DESCRIPTION -->
+
+<!-- Please give a few sentences describing the overall goals of the pull request. -->
+<!-- Give enough details to make the improvement and changes of the PR understandable  -->
+<!-- to both developers and tech-savy users. The description can contain but is not limited to: -->
```

Comment:
I just notice that the list below is (or became) inconsistent, the first three items have a different format and type as the others the later or more questions / check maybe we should rephrase the first three to the same style.

Also we should maybe change the sentence "The description can contain but is not limited to" to something like, please check whether your contribution against the following checklist of something similar.